tests/testsuite/test.rs

```diff
@@ -3967,6 +3967,58 @@ fn test_dep_with_dev() {
         .run();
 }
 
+#[cargo_test]
+fn doctest_target_for_host_target_only() {
```

Comment:
This test seems to essentially duplicate the `no_cross_doctests` test. Perhaps instead o adding this test, `no_cross_doctests` could be extended to check for the flag?  Essentially change [this line](https://github.com/rust-lang/cargo/blob/9fa2bba6791f9df06a13dbfe18bf7add9b9b2d1b/tests/testsuite/cross_compile.rs#L595) to include `--verbose` and extend the `with_stdout` to check for `rustdoc [..]--target {target}` (and maybe also include `.with_stdout_contains("test [..] ... ok")`).

Reply:
Updated. Thanks for the help!